pkg/task/backup_raw.go

```diff
@@ -26,15 +26,20 @@ const (
 	flagEndKey           = "end"
 )
 
-// BackupRawConfig is the configuration specific for backup tasks.
-type BackupRawConfig struct {
+// RawKvConfig is the common config for rawkv backup and restore.
+type RawKvConfig struct {
 	Config
 
 	StartKey []byte `json:"start-key" toml:"start-key"`
 	EndKey   []byte `json:"end-key" toml:"end-key"`
 	CF       string `json:"cf" toml:"cf"`
 }
 
+// BackupRawConfig is the configuration specific for backup tasks.
+type BackupRawConfig struct {
```

Comment:
Maybe `RawKVConfig` here is enough? `BackupRawConfig` seems nothing special

Reply:
I left it here just trying to make a good semantics